docs/libcurl/symbols-in-versions

```diff
@@ -599,6 +599,8 @@ CURLOPT_SSL_SESSIONID_CACHE     7.16.0
 CURLOPT_SSL_VERIFYHOST          7.8.1
 CURLOPT_SSL_VERIFYPEER          7.4.2
 CURLOPT_SSL_VERIFYSTATUS        7.41.0
+CURLOPT_SSL_VERIFYSTATUS_FUNCTION        7.64.2
+CURLOPT_SSL_VERIFYSTATUS_DATA        7.64.2
```

Comment:
This won't actually go into 7.64.2 (since there won't be any such version), but we can update these version numbers once we know where it'll end up.